recipe/build.sh

```diff
@@ -4,17 +4,62 @@ set -euxo pipefail
 
 if [[ "${target_platform}" == osx-* ]]; then
   export LDFLAGS="${LDFLAGS} -lz -framework CoreFoundation -Xlinker -undefined -Xlinker dynamic_lookup"
+  # Remove stdlib=libc++; this is the default and errors on C sources.
+  export CXXFLAGS=${CXXFLAGS/-stdlib=libc++}
```

Comment:
This is probably functional but double slashes allows all `-stdlib=libc++` patterns to be removed. So if somehow `CXXFLAGS` ended up looking something like `-stdlib=libc++ -ggdb3 -O0 -stdlib=libc++` then both would be removed.
```suggestion
  # Remove stdlib=libc++; this is the default and errors on C sources.
  export CXXFLAGS=${CXXFLAGS//-stdlib=libc++}
```